### ISSUE TITLE
Option to force HTTPS redirect

### ISSUE THREAD

**Issue by straussmarkus** [mschlenstedt/Loxberry#1482]
I am glad that we now can use custom certificates for HTTPS. I think it would be good to add an option that redirects from HTTP to HTTPS so that you don't accidentally access without HTTPS.

**Comment by christianTF** [comment]
Needs to be discussed.
MS Gen.1 cannot use https, also other IoT devices may fail. 

Needs to be made _very clear_  that this can break communication to plugins in some circumstances.

**Comment by Ihon75** [comment]
I would agree with straussmarkus to add an option for a forced redirect to HTTPS.
A good explenation can still be part of the switch or even a small logic that grays out the switch if a MS Gen.1 connection is there

**Comment by mschlenstedt** [comment]
Will not be implemented - sorry. Risk is too high for me.
